### DISCUSSION TITLE
How to disable linkerd at container level in a pod

### DISCUSSION THREAD

**Discussion by saikrishnakumarreddy** [linkerd/linkerd2#12439] [Q&A] [answered]
How to disable linkerd at container level in a pod ? lets say i have 5 container in my pod and i want only one container to use linkerd side car proxy.

**Comment by wmorgan** [comment, marked as answer]
Probably the only option is to use use skip-inbound-ports and skip-outbound-ports to bypass the proxy for the specific ports those containers use.
